Dockerfile

```diff
@@ -30,4 +30,6 @@ WORKDIR /workspace
 COPY workspace/codewars_reporter.c .
 COPY workspace/tests.c .
 
+ENV QEMU_CPU=rv64,v=on,vext_spec=v1.0,zbkx=on,zk=on,zks=on
```

Comment:
Let's move this to `bin/run` so the image doesn't require QEMU 7.1 on host by default.

Reply:
Done :heavy_check_mark: Thanks for the suggestion